test/config/launch-config.json

```diff
@@ -50,6 +50,7 @@
           "port": 31200,
           "name": "alice",
           "flags": [
+            "-lethereum-light-client=trace",
```

Comment:
This would be the beacon client surely?

Reply:
I reverted this in https://github.com/Snowfork/snowbridge/pull/653/commits/45db3ef5727c1fa56e7c5b0f12b387bce7f0cacd, accidentally commit.